packages/template-chakra-storefront/src/components/_app/hooks/use-app-analytics.js

```diff
@@ -7,7 +7,7 @@
 
 import {useEffect} from 'react'
 import {useLocation} from 'react-router-dom'
-import useActiveData from '../../../hooks/use-active-data'
+import {useActiveData} from '../../../hooks/use-active-data'
```

Comment:
with all the hooks is registed in hooks/index.js. I think we can do this for all hook import
```
import {useActiveData} from '../../../hooks
```